lib/i18n/hygiene/keys_with_matched_value.rb

```diff
@@ -6,7 +6,7 @@ class KeysWithMatchedValue
 
       def initialize(regex, i18n_wrapper = nil, reject_keys: nil)
         @regex = regex
-        @i18n = i18n_wrapper || I18n::Hygiene::Wrapper.new
```

Comment:
It's interesting that the build didn't fail after the `I18n::Hygiene::Wrapper` constructor changed. I'm guessing that's down to the unit specs for these classes using a double rather than a real `I18n::Hygiene::Wrapper`?